ext/hal/st/stm32cube/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ if(CONFIG_SOC_SERIES_STM32F1X)
   zephyr_sources(stm32f1xx/drivers/src/stm32f1xx_hal.c)
   zephyr_sources(stm32f1xx/drivers/src/stm32f1xx_hal_rcc.c)
   zephyr_sources_ifdef(CONFIG_CLOCK_CONTROL_STM32_CUBE stm32f1xx/drivers/src/stm32f1xx_ll_utils.c)
+  zephyr_sources_ifdef(CONFIG_SPI_STM32 stm32f1xx/drivers/src/stm32f1xx_ll_spi.c)
```

Comment:
This is not necessary anymore after PR #4646 is merged. Works for me without.

Reply:
Fixed